meinberlin/react/livequestions/QuestionBox.jsx

```diff
@@ -237,68 +232,43 @@ export default class QuestionBox extends React.Component {
           aria-hidden="true"
         >
           {this.props.hasAskQuestionsPermission &&
-            <div className="container">
-              <div className="offset-lg-2 col-lg-8">
-                <QuestionForm
-                  restartPolling={this.restartPolling}
-                  category_dict={this.props.category_dict}
-                  questions_api_url={this.props.questions_api_url}
-                  privatePolicyLabel={this.props.privatePolicyLabel}
-                />
-              </div>
+            <QuestionForm
+              restartPolling={this.restartPolling}
+              category_dict={this.props.category_dict}
+              questions_api_url={this.props.questions_api_url}
+              privatePolicyLabel={this.props.privatePolicyLabel}
+            />}
+          <Filters
+            categories={this.props.categories}
+            currentCategory={this.state.category}
+            setCategories={this.setCategory.bind(this)}
+            orderedByLikes={this.state.orderedByLikes}
+            toggleOrdering={this.toggleOrdering.bind(this)}
+            displayOnShortlist={this.state.displayOnShortlist}
+            displayNotHiddenOnly={this.state.displayNotHiddenOnly}
+            toggleDisplayOnShortlist={this.toggleDisplayOnShortlist.bind(this)}
+            toggledisplayNotHiddenOnly={this.toggledisplayNotHiddenOnly.bind(this)}
+            isModerator={this.props.isModerator}
+          />
+          {this.props.isModerator &&
+            <div className="block">
+              <a className="btn btn--light" rel="noopener noreferrer" href={this.props.present_url} target="_blank">
+                <span className="fa-stack fa-1x">
+                  <i className="fas fa-tv fa-stack-2x" aria-label="hidden"> </i>
+                  <i className="fas fa-arrow-up fa-stack-1x" aria-label="hidden"> </i>
+                </span>
+                {displayStr}
+              </a>
             </div>}
-          <div>
-            <div className="container">
-              <div className="offset-lg-2 col-lg-8">
-                <InfoBox
```

Comment:
I removed the `<InfoBox />` component as it is no longer in the design and the component is only used here.

It originally looked like this:

<img width="564" alt="Screenshot 2024-11-13 at 09 24 11" src="https://github.com/user-attachments/assets/881d553a-0a4f-4f24-99b0-69a415b65e97">

Reply:
This might be part of a later story and not in the design because it's [maybe? @hom3mad3] a view for moderators only.

Reply:
I've reverted to using it: maybe we can remove it if needed in subsequent PR.